.travis.yml

```diff
@@ -1,8 +1,8 @@
 sudo: false
 node_js:
-- "4"
```

Comment:
I realize jsdom doesn't support Node 4, but Marko 4 still does. This means we will either have to strengthen our linting rules to ensure we aren't breaking Node 4, or this is a major version.

Reply:
Yeah, Michael and I were talking about this. I agree it should be under a major. We were debating if it would make sense to just release a Marko v5 (and start releasing mostly non breaking majors more often) or I may just switch back to jsdom 9. We can hash this out tomorrow morning though.